packages/shared/components/Profile.svelte

```diff
@@ -31,20 +34,23 @@
 </script>
 
 <div class="flex items-center justify-center w-24">
-    <div class="flex flex-col justify-between items-center space-y-3">
-        <div
-            on:click={() => handleOnClick()}
-            class="h-20 w-20 {bgColor
-                ? `bg-${bgColor}-500`
-                : ''} rounded-full font-bold text-center flex items-center justify-center {classes}"
-        >
-            {#if slots}
-                <slot />
-            {:else}
-                <Text type="h3" classes="text-white">{getInitials()}</Text>
-            {/if}
+    <div class="flex flex-col justify-between items-center">
+        <div class="relative mb-3">
+            <div
+                on:click={() => handleOnClick()}
```

Comment:
this is artefact of what was already there, but we don't need to pass in the handler like this if it doesn't have any params. 

on:click is expecting a function and not a function call, hence why if we need to pass in a param we do () => functionName(params), But if we are just passing the function we can write it like:
```suggestion
                on:click={handleOnClick}
```

Reply:
Thanks for the explanation. I'll keep this in mind for other instances I come across in Firefly.